JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.ts

```diff
@@ -105,14 +105,15 @@ export class AppComponent {
     items: this.lineHeights,
     selectedItemKey: this.lineHeight,
     onSelectionChanged: (): void => {
-      notify('Line height value has been changed!');
+      this.onSelectionChanged('Line Height');
     },
   };
 
   fontFamilyOptions = {
     placeholder: 'Font',
     displayExpr: 'text',
     dataSource: new DataSource(this.fontFamilies),
+    onItemClick: this.onFontFamilyClick,
```

Comment:
Let's use arrow function to not lose a context

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2062/commits/3b64bba423f7a9d79322fbcd48590031fc94655b